src/config.js

```diff
@@ -63,3 +63,9 @@ module.exports.retrieveUserConfig = function(pjson, name) {
 
   return null;
 };
+
+module.exports.retrieveUserConfig = retrieveUserConfig;
+module.exports.buildConfig = buildConfig;
+module.exports.newsFragmentsUserConfig = buildConfig(
+  retrieveUserConfig(pjson, "@grupoboticario/news-fragments")
+);
```

Comment:
Não sei se vai funcionar, porque ele pode pegar o nosso `pjson` em qualquer projeto

Reply:
🤔 No plugin, o `this.baseConfig` fazia isso.